<launch>
  <!-- Tester for biarritz-->
  <!-- It  includes visualization nodes -->
    
  <!-- Launch kompai nodes -->
  <!--  <include file="$(find ros_pure)/launch/ros_pure.launch" />  -->

  <!-- Launch RFID Node -->
  <node name="rfid_reader" pkg="rfid_node" type="rfid_reader_ros.py"/>

  <!-- We need a map published under /map topic... -->

  <!-- Load regions descriptions used by all object trackers -->
  <rosparam command="load" file="$(find rfid_grid_map)/config/ROBOSOFT3.yaml"/>


  <!-- Launch the grid map and visualization for a tag -->
  <include file="$(find rfid_grid_map)/launch/grid.launch">
    <!-- all vars that included.launch requires must be set -->
    <arg name="tagID" value="390000010000000000000018" />
    <arg name="object" value="glasses" />
    <arg name="saveRoute" value="$(find rfid_grid_map)/save/ROBO/" />    
    <arg name="loadGrids" value="false" />        
  </include>
  

  <!-- Launch the grid map and visualization for a tag -->
  <include file="$(find rfid_grid_map)/launch/grid.launch">
    <!-- all vars that included.launch requires must be set -->
    <arg name="tagID" value="390101110000000000000009" />
    <arg name="object" value="wallet" />
    <arg name="saveRoute" value="$(find rfid_grid_map)/save/ROBO/" />    
    <arg name="loadGrids" value="false" />        
  </include>
  
 <!-- Launch ROL Node -->
  <!-- <node name="rol_srv_node" pkg="rol_server" type="rol_server_node.py"/> -->

  <!-- Launch RViz with the demo configuration -->
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find rfid_grid_map)/rviz/rfid_gridmap_test.rviz" />
</launch>

